SF iOS/SF iOS/Helpers/UserNotification/BackgroundFetcher.m

```diff
@@ -48,9 +48,14 @@ - (void)didChangeDataSourceWithInsertions:(nullable NSArray<NSIndexPath *> *)ins
     NSInteger currentBadgeCount = [[UIApplication sharedApplication] applicationIconBadgeNumber];
 
     for (NSIndexPath *update in updates) {
-        NSString *contentTitle = NSLocalizedString(@"Coffee Event changed",
-                                                   @"notification title for changed events");
         Event *event = [self.backgroundDataSource eventAtIndex:[update row]];
+        // Prevent notifications if changes, e.g. images and URLs, are made to past events
+        if ([[event endDate] compare:[NSDate date]] == NSOrderedAscending) {
+            continue;
+        }
+        
+        NSString *contentTitle = NSLocalizedString(@"Coffee Event changed",
```

Comment:
I _think_ it's good, but we also have this: 

```
- (BOOL)isInFuture {
    return [self compare:[NSDate new]] == NSOrderedDescending;
}
```

in `NSDate+Utilities.m". Would you mind using that?